mscore/capella.h

```diff
@@ -570,7 +570,7 @@ class ChordObj : public BasicDurationalObj, public NoteObj {
    public:
       enum class StemDir : signed char { DOWN = -1, AUTO = 0, UP = 1, NONE = 3 };
       BeamMode beamMode;
-      char notationStave;
+      signed char notationStave;
```

Comment:
see the first comparison in https://github.com/ericfont/MuseScore/blob/unspecified-chars/mscore/capella.cpp#L1835 <pre>notationStave >= -1</pre> which is always true when notationStave is an unspecified char on arm.  Of course it is in an assert statement, which explains why arm compiler didn't complain when making release in the past.